### PR TITLE
Add Arch specific requirements installation guide in README.md

### PR DESCRIPTION
**Pull request by mkoushan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +57,8 @@ sudo dnf install ghdl gtkwave
     ```
 
     **Note:** The AppImage does not provide a GUI, so double-clicking it does nothing.
-
+    
+    **Note:** For the Arch make sure to install [AppImageLauncher<sup>AUR</sup>](https://aur.archlinux.org/packages/appimagelauncher) first.
```
Comment (machitgarha) [inline README.md:61]: Parvaj is terminal-based, so no need for AppImageLauncher.

File: README.md [hunk 2]
```diff
@@ -122,6 +128,7 @@ Tested platforms include:
 
 -   Fedora 28+
 -   Ubuntu 18.04+
+-   Arch 6.2.2-zen2-1-zen+
```
Comment (machitgarha) [inline README.md:131]: Although not documented, but tested platforms are those inside which I manually tested the resulting AppImage, using Docker containers.

So did you test it too?

**Review by machitgarha — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please fix the issues in order to get merged.

File: README.md (hunk 1 above)
Comment (mkoushan) [reply README.md:61]: For the AppImage way to install and run, it is needed.
I tested without that package and I got this:
```bash
exec: Failed to execute process '/home/koushan/.local/bin/parvaj': The file exists and is executable. Check the interpreter or linker?
```

Comment (machitgarha) [reply README.md:61]: Ah. That's unfortunate. Had you installed required libraries for running AppImages (like fuse)?

BTW, I'll test it in an Arch Docker container.

Comment (machitgarha) [reply README.md:61]: I've checked it. At least for Arch Linux's Docker container, there is no need for having AppImageLauncher.

Generally, AppImages are self-contained, so they doesn't rely on things like AppImageLauncher.

**Pull request merged**